docs/userguide/src/portingguide/howto/nogc.md

```diff
@@ -110,6 +112,29 @@ We recommend going through the [list of constants in the documentation](https://
 ## MMTk initialization
 Now that we have most of the boilerplate set up, the next step is to initialize MMTk so that we can start allocating objects.
 
+In short, MMTk uses the builder pattern. The binding needs to create an [`MMTKBuilder`](https://docs.mmtk.io/api/mmtk/struct.MMTKBuilder.html),
+create an [`MMTK`](https://docs.mmtk.io/api/mmtk/mmtk/struct.MMTK.html) instance from the builder, and then initialize MMTk's collection
+when the runtime system is ready for GCs.
+The following steps describes details. In an actual binding implementation, the binding may choose to combine several
+steps into one function call to make things simpler.
+
+1. Create an `MMTKBuilder` using [`MMTKBuilder::new()`](https://docs.mmtk.io/api/mmtk/struct.MMTKBuilder.html#method.new). You can set
+   runtime options via [`set_option()`](https://docs.mmtk.io/api/mmtk/struct.MMTKBuilder.html#method.set_option) for things like
```

Comment:
It's worth mentioning that it is possible to set options directly using statements like `builder.options.threads.set(4);`, and perhaps it is better to mention that before mentioning `set_option` which parses string.  Wenyu once tried to convert integer to strings just for passing the string to `set_option` because he didn't know it was possible to set options using Rust values directly.

Reply:
Done.

Reply:
Did you forget to commit the change? The document is not changed.

Reply:
Yeah. Sorry. I had the changes on one machine, and switched to another machine to reproduce the test bug and forgot to push the changes on the first machine. I pushed the changes now.